Server/ReasnAPI/ReasnAPI/Services/EventService.cs

```diff
@@ -216,17 +191,43 @@ public EventDto GetEventById(int eventId)
         return eventDto;
     }
 
-    public EventDto GetEventBySlug(string slug)
+    public Event GetEventBySlug(string slug)
     {
         var eventToReturn = context.Events.Include(e => e.Tags).Include(e => e.Parameters).FirstOrDefault(e => e.Slug == slug);
         if (eventToReturn is null)
         {
             throw new NotFoundException("Event not found");
         }
 
-        var eventDto = eventToReturn.ToDto();
 
-        return eventDto;
+        return eventToReturn;
+    }
+
+    public IEnumerable<ParticipantDto> GetEventParticipantsBySlug(string slug)
+    {
+        var eventToReturn = context.Events.Include(e => e.Tags).Include(e => e.Parameters).Include(e => e.Participants).FirstOrDefault(e => e.Slug == slug);

```

Comment:
Do we need to include here Tags and Parameters? We only need Participants

Reply:
changed